packages/compress/zstd/package.mk

```diff
@@ -10,7 +10,6 @@ PKG_URL="https://github.com/facebook/zstd/releases/download/v${PKG_VERSION}/${PK
 PKG_DEPENDS_HOST="gcc:host ninja:host"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_LONGDESC="A fast real-time compression algorithm."
+PKG_TOOLCHAIN="meson"
 
-configure_package() {
-  PKG_CMAKE_SCRIPT="${PKG_BUILD}/build/cmake/CMakeLists.txt"
-}
+PKG_MESON_SCRIPT="$(get_build_dir ${PKG_NAME})/build/meson/meson.build"
```

Comment:
If you use:
```
configure_package() {
  PKG_MESON_SCRIPT="${PKG_BUILD}/build/meson/meson.build"
}
```
it avoids a fork (or maybe two), and would be the better solution IMHO.

Reply:
Sure, I like this way because it's cleaner but your reasoning make sense.